### PR TITLE
Add missing job id

### LINKED ISSUE #4819: Missing github actions default environment variable
### What is the current behavior?

[Page](https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables) is missing GITHUB_JOB
The equivalent context is on this [page](https://github.com/github/docs/blob/main/content/actions/reference/context-and-expression-syntax-for-github-actions.md)

### What changes are you suggesting?

Add the missing variable.
You can see it is present and public by looking at this [code](https://github.com/actions/toolkit/blob/4bf916289e5e32bb7d1bd7f21842c3afeab3b25a/packages/github/src/context.ts#L43)

### PR DESCRIPTION
**Pull request by tonyhallett**
### Why:

Closes #4819 

### What's being changed:
Added row in table for missing github actions default environment variable.

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
@tonyhallett Thanks so much for opening an issue and a PR! I'll get this triaged for review ⚡

File: content/actions/reference/environment-variables.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ We strongly recommend that actions use environment variables to access the files
 | `GITHUB_WORKFLOW` | The name of the workflow. |
 | `GITHUB_RUN_ID` | {% data reusables.github-actions.run_id_description %} |
 | `GITHUB_RUN_NUMBER` | {% data reusables.github-actions.run_number_description %} |
+| `GITHUB_JOB` | The [job_id](/actions/reference/workflow-syntax-for-github-actions#jobsjob_id) of the current job |
```
Comment (skedwards88) [inline content/actions/reference/environment-variables.md:57]:
```suggestion
| `GITHUB_JOB` | The [job_id](/actions/reference/workflow-syntax-for-github-actions#jobsjob_id) of the current job. |
```

**Review by skedwards88 — APPROVED** [review]
:shipit: Thanks for finding this missing variable and adding it to the docs! I'll get your changes merged down.

**Review by jace833 — APPROVED** [review]
jace05@tuta.io

**Review by jace833 — APPROVED** [review]

**Comment by Ao-2** [comment]
[]()

**Review by jace833 — COMMENTED** [review]
[moderator removed content]

**Pull request merged**